subsys/bluetooth/services/cgms/Kconfig.cgms

```
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig BT_CGMS
+	bool "Enable GATT continuous glucose service"
+	select BT_NRF_SERVICES
```

Please tag this module as experimental

```suggestion
        select EXPERIMENTAL
	select BT_NRF_SERVICES
```